.rubocop.yml

```diff
@@ -247,4 +247,4 @@ Style/ClassAndModuleChildren:
   Enabled: false
 
 RSpec/MultipleExpectations:
-  Max: 3
+  Max: 7
```

Comment:
This is too big and defeat the purpose of the scope

Reply:
Would changing it to 5 do (that would be enough to not violate current tests) or should I revert it back to 3?

Reply:
@tachyons